packages/components/organisms/f-checkout/src/tenants/en-NZ.js

```diff
@@ -1,4 +1,223 @@
+                message: 'We’re sorry, but the restaurant is not currently taking orders. Please choose another restaurant to order from.',
+                buttonText: 'Ok'
+            },
+
+            SERVICE_TYPE_UNAVAILABLE: {
+                title: '{serviceType} unavailable',
+                message: 'We’re sorry, but the restaurant is not currently taking {serviceType} orders.',
+                buttonText: 'Back to order'
+            },
+
+            ADDITIONAL_ITEMS_REQUIRED: {
+                title: 'Item(s) cannot be ordered',
+                message: 'We’re sorry, but the item(s) you are trying to order require other items before you can continue. Please amend your order. ',
+                buttonText: 'Back to order'
+            },
+
+            ITEMS_UNORDERABLE: {
+                title: 'Item(s) unavailable',
+                message: 'We’re sorry, but some item(s) you wanted to order are no longer available. Please check your order before continuing.',
+                buttonText: 'Back to order'
+            },
+
+            FULFILMENT_TIME_UNAVAILABLE: {
+                title: 'Time unavailable',
+                message: 'We’re sorry, but the time you have selected is no longer available. Please select another time.',
+                buttonText: 'Ok'
+            },
+
+            LOCATION_UNDELIVERABLE: {
+                title: 'Cannot deliver to address',
+                message: 'We’re sorry, but the restaurant does not currently deliver to this address. Please choose to order via collection or from another restaurant.',
+                buttonText: 'Ok'
+            },
+
+            TABLE_IDENTIFIER_REQUIRED: {
+                title: 'Table number/name is required',
+                message: 'It appears you didn’t tell us how to find you in the restaurant. Please enter your table number and try again.',
+                buttonText: 'Ok'
+            },
+
+            DEFAULT_CHECKOUT_ISSUE: {
+                title: 'Something went wrong',
+                message: 'Please try again',
+                buttonText: 'Ok'
+            },
+
+            DuplicateOrder: {
+                title: 'This order has not been placed',
+                message: 'It’s the same as the one you made a moment ago and we want to check it’s not a mistake. Wait 15 mins or change an item in your order',
+                buttonText: 'Close',
+                buttonTextPrimary: 'View my orders'
+            }
+        }
+    },
+
+    warningMessages: {
+        preOrder: {
+            title: 'Please note, this is a preorder',
+            body: 'Please check the day and time above'
+        }
+    },
+
+    buttonText: 'Go to payment',
+
+    userNote: {
+        delivery: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn\'t work). Do not include details about any allergies here.',
```

Comment:
As above, proper quote marks here? Or not desirable as an example of proper user input?
```suggestion
            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn’t work). Do not include details about any allergies here.',
```